apis/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/metal3-io/baremetal-operator/apis
 
-go 1.21
+go 1.22.8
```

Comment:
These vulns are not called, so they're irrelevant. In release branch we should not bump go without good reason.